parser/statement_parsing.go

```diff
@@ -170,7 +193,11 @@ func (p *Parser) parseReturnStatement() *ast.ReturnStatement {
 func (p *Parser) parseExpressionStatement() *ast.ExpressionStatement {
 	stmt := &ast.ExpressionStatement{Token: p.curToken}
 
-	stmt.Expression = p.parseExpression(LOWEST)
+	precedence := 1
```

Comment:
`LOWEST` should be literally the lowest.
I think a better way is to name your constant `CALL_WITHOUT_PAREN`, which is very verbose but also clear. And rename `LOWEST` to `NORMAL`.